ansible/roles/master/templates/kubelet.j2

```diff
@@ -8,10 +8,10 @@ KUBELET_ADDRESS="--address=0.0.0.0"
 # KUBELET_PORT="--port=10250"
 
```

Comment:
Note that these changes need to work for ha and non-ha scenario's. If ha works without removing lines 11 and 14, i suggest adding back in.